source/eprosima-suite/eprosima_suite.rst

```diff
@@ -0,0 +1,152 @@
+.. _docker_usage:
+
+Docker Installation and Usage
+-----------------------------
+
+To run this container you need Docker installed. From a terminal run
+
+.. code-block:: bash
+
+    $ sudo apt install docker.io
+
+To load this image into your Docker repository, from a terminal run
+
+.. code-block:: bash
+
+ $ docker load -i ubuntu-eprosima-suite:<FastDDS-Version>.tar
+
+You can run this Docker container as follows
+
+.. code-block:: bash
+
+ $ xhost local:root
+ $ docker run -it --privileged -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix \
+ ubuntu-eprosima-suite:<FastDDS-Version>
+
+From the resulting Bash Shell you can run each feature.
+
+.. _fast_dds_suite_examples:
+
+Fast DDS Examples
+-----------------
+
+Included in this Docker container is a set of binary examples that showcase several functionalities of the
+Fast DDS libraries. These examples' path can be accessed from a terminal by typing
```

Comment:
I'd say the Fast DDS is 1 library. It is true that it depends on Fast CDR, but the end user does not interact with it directly, so they only use one library
```suggestion
Fast DDS library.
These examples' path can be accessed from a terminal by typing
```